ts/features/itwallet/screens/presentation/remote/pid/ItwPrRemotePidDataScreen.tsx

```diff
@@ -76,116 +54,100 @@ const ItwPrRemotePidDataScreen = () => {
     });
   };
 
-  const RpPreviewView = ({ pid }: { pid: StoredCredential }) => (
-    <SafeAreaView edges={["bottom", "left", "right"]} style={IOStyles.flex}>
+  const ContentView = ({ pid }: { pid: StoredCredential }) => {
+    const claims = parseClaims(pid.parsedCredential);
+    const requiredClaims = claims.map(
+      claim =>
+        ({
+          claim,
+          source: pid.credentialType
+        } as RequiredClaim)
```

Comment:
**suggestion (non-blocking)**: I'm not sure, but I think we can remove the `as RequiredClaim`

Reply:
Fixed in [ceaa369](https://github.com/pagopa/io-eudiw-app/pull/40/commits/ceaa369f2b20386b8ec60017e2d64d15b22f6e18), thanks.